crypto/src/key/mod.rs

```diff
@@ -64,10 +65,16 @@ impl From<RistrettoSignatureError> for SignatureError {
 
 impl PrivateKey {
     pub fn new(key_kind: KeyKind) -> (PrivateKey, PublicKey) {
```

Comment:
Can you rename this to `new_from_entropy`?

Reply:
Updated, thanks.